apl-updater/src/test/java/com/apollocurrency/aplwallet/apl/updater/SimpleTransaction.java

```diff
@@ -245,5 +297,29 @@ public int getECBlockHeight() {
     public long getECBlockId() {
         return 0;
     }
+
+
+    public boolean attachmentIsDuplicate(Map<TransactionType, Map<String, Integer>> duplicates, boolean atAcceptanceHeight) {
```

Comment:
Why do we need this method here?

Reply:
That method came from **Transaction Interface**. I would like all three classes like : _SimpleTransaction, TransactionImpl, UnconfirmedTransaction_  to be lined up with **Transaction Interface**. If implementation code is not applicable here, we can just throw some exception here in proper class.

Reply:
Implementation is not required here. We can also set this methos as default in Transaction interface

Reply:
Done and pushed.